### ISSUE TITLE
Avoid noisy "Duplicate file in archive picking first occurrence" in container_layer when tars contain duplicate files

### ISSUE THREAD

**Issue by u-quark** [bazelbuild/rules_docker#827]
While creating an image layer with tars the warning for duplicate files "Duplicate file in archive picking first occurrence" can be very noisy. In our case the tars contain identical files so this should not really be an issue. Also it is not trivial to create disjoint tars. It would be nice if there was a way to silence this warnings.

I opened a related issue on https://github.com/bazelbuild/bazel/issues/8260 but there should be a way to work around this within `rules_docker`.

### Steps to reproduce

```
$ mkdir tmp
$ touch tmp/somefile
$ tar cf a.tar tmp/
$ tar cf b.tar tmp/
$ cat BUILD.bazel 
load("@io_bazel_rules_docker//container:container.bzl", "container_layer")

container_layer(
  name = "foo",
  tars = [":a.tar", ":b.tar"],
)
$ bazel build //:foo
INFO: Invocation ID: 34b3eb46-bb93-4fc3-bd5d-2d272b24e620
INFO: Analysed target //:foo (1 packages loaded, 3 targets configured).
INFO: Found 1 target...
INFO: From ImageLayer foo-layer.tar:
Duplicate file in archive: ./tmp/somefile, picking first occurrence   <<<<<<<<<<
Target //:foo up-to-date:
  bazel-bin/foo-layer.tar
INFO: Elapsed time: 1.243s, Critical Path: 0.51s
INFO: 1 process: 1 linux-sandbox.
INFO: Build completed successfully, 4 total actions
```

**Comment by smukherj1** [comment]
Hi @u-quark,

Thanks for reporting the issue. Looking at the code, I see we are calling the archive.py shipped with Bazel as per the dependency [here](https://github.com/bazelbuild/rules_docker/blob/f737dd01fa2255feb84c87adf442e2d32824f420/container/BUILD#L92). Since, the message is emitted from the archive.py in Bazel, we'll need the archive to accept a directive asking it to silence such errors. Thus, I think we need https://github.com/bazelbuild/bazel/issues/8260 to be resolved and the change to be available in a new Bazel release before we can change rules_docker to not emit the message.

**Comment by nlopezgi** [comment]
Closing as the fix for this is not in this repo, let me know if anything is pending to reopen.
